client/src/partials/budget/new/new_budget.html

```diff
@@ -0,0 +1,199 @@
+              <label style="display:block;">
+                <input type="radio" name="session.option" 
+                  class="select-csv" 
+                  ng-model="session.option" value="csv" ng-checked="true"><b>{{ "BUDGET.NEW.IMPORT_CSV" | translate }}</b>
+              </label>
+              <label style="display:block;">
+                <input type="radio" name="session.option" 
+                  ng-model="session.option" value="manual"><b>{{ "BUDGET.NEW.MANUAL" | translate }}</b>
+              </label>
+            </div>
+          </div>
+
+          <div ng-switch on="session.option" class="well">
+            <div ng-switch-when="csv" class="form-inline">
+              <h4><b><i class="glyphicon glyphicon-file"></i> {{ "BUDGET.NEW.IMPORT_CSV" | translate }}</b></h4>
+              <div class="form-group">
+                <label>{{ "BUDGET.ANALYSIS.SELECT_PERIOD" | translate }}</label>
+              </div>
+              
+              <select 
+                class="form-bhima" 
+                name="period"
+                ng-if="session.option=='csv'"
+                ng-model="config.period" 
+                ng-options="p as formatPeriod(p) for p in session.periods" 
+                ng-disabled="!config.fiscal_year_id" 
+                required>
+              </select>
+              <br>
+              <button 
+                ngf-select="uploadFile($file)"
+                ngf-pattern="'.csv'" 
+                accept=".csv" 
+                class="btn btn-sm btn-success"
+                ng-disabled="!config.period.id">
+                <i class="glyphicon glyphicon-upload"></i> {{ "BUDGET.NEW.SEND_FILE" | translate }}
+              </button>
+            </div>
+
+            <div ng-switch-when="manual">
+              <div find-account 
+                on-submit="submitAccount" 
+                on-reset="resetAccountSearch" 
+                where="accountWhere" 
+                enable-reset></div>
+            </div>
+          </div>
+
+        </form>
+
+      </div>
+    </div>
+
+    <div ng-switch-when="edit">
+      <div class="row" ng-if="!session.found">
+        <div class="col-sm-offset-3 col-xs-6">
+          <div class="center">
+            <h3>{{ session.no_data_msg }}</h3>
+            <h3>{{ "BUDGET.EDIT.CREATE_QUESTION" | translate }}</h3>
+            <h3>
+              <button class="btn btn-sm btn-default" ng-click="createBudget()">
+                <span class="glyphicon glyphicon-plus-sign"></span> {{ "BUDGET.EDIT.CREATE" | translate }}
+              </button>
+            </h3>
+          </div>
+        </div>
+      </div>
+
+      <div class="row" ng-if="session.found">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <div class="col-xs-8">
+              <form name="budget">
+                <table class="table table-condensed form-bhima-border" summary="{{ 'EDIT.BUDGET.EDIT' | translate }}">
```

Comment:
Table `summary` seems like a useful attribute, but according to the [Mozilla Developer Network](https://developer.mozilla.org/en/docs/Web/HTML/Element/table), it is deprecated.  Here's what the Mozilla developers recommend:
> Usage Note: Do not use this attribute, as it has been deprecated. Instead, use one of these way of describing a table:
In prose, surrounding the table (this is the less semantic-conveying way of doing it).
In the table's `<caption>` element.
In a `<details>` element, inside the table's `<caption>` element.
Include the `<table>` element in a `<figure>` element and add the description in prose next to it.
Include the `<table>` element in a `<figure>` element and add the description in prose inside a `<figcaption>` element.
Adjust the table so that such description is no more needed, by using `<th>` and `<thead>` elements for example.
> -- Mozilla Developer Network